qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1517A.py

```diff
@@ -21,6 +21,7 @@
 
 if TYPE_CHECKING:
     from .KeysightB1500_base import KeysightB1500
+    from .KeysightB1511B import B1511B
```

Comment:
there's not need for this anymore because B1511B inherits from B1517A, this places where B1517A is needed, B1511B is allowed automatically.